TCAT/Controllers/FavoritesTableViewController.swift

```diff
@@ -81,17 +103,29 @@ class FavoritesTableViewController: UITableViewController {
             dismiss(animated: true)
         }
     }
+}
 
-    // MARK: - Table view data source
-    override func numberOfSections(in tableView: UITableView) -> Int {
-        return 1
+// MARK: - Table view data source
+extension FavoritesTableViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return resultsSection.items.count
     }
 
-    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return resultsSection.items.count
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: Constants.Cells.placeIdentifier, for: indexPath) as! PlaceTableViewCell
+        cell.place = resultsSection.items[indexPath.row]
+        cell.layoutSubviews()
```

Comment:
what is `layoutSubviews()` for? the constraints for your views should be done in the `init` of the table view cell. also if you want to configure the cell based on the `place` model, you should make a `configure` function